apps/test-suite/stories/1-components/BlurView/BlurViewScreen.js

```diff
@@ -0,0 +1,101 @@
+import { BlurView } from 'expo';
+import React from 'react';
+import { View, Image, StyleSheet, Text } from 'react-native';
+
+import UIExplorer, {
+  AppText,
+  Code,
+  Description,
+  DocItem,
+  Section,
+  storiesOf,
+} from '../../ui-explorer';
+
+const image = {
+  uri:
+    'https://www.tesla.com/tesla_theme/assets/img/_vehicle_redesign/roadster_and_semi/roadster/hero-portrait@2.jpg',
+};
+const Screen = () => (
+  <View>
+    <UIExplorer title="Blur View">
+      <Description>
+        <AppText>
+          A React component that renders a native blur view on iOS and falls back to a
+          semi-transparent view on Android. A common usage of this is for navigation bars, tab bars,
+          and modals.
+        </AppText>
+      </Description>
+      <DocItem
+        name="Importing the module"
+        example={{
+          code: `// in managed apps:
+import { BlurView } from 'expo';
```

Comment:
We are moving to `...from 'expo-blur'` everywhere, just use the unimodule import